iceoryx_binding_c/include/iceoryx_binding_c/internal/c2cpp_enum_translation.hpp

```diff
@@ -24,7 +24,8 @@
 
 namespace c2cpp
 {
-iox::popo::SubscriberTooSlowPolicy subscriberTooSlowPolicy(const ENUM iox_SubscriberTooSlowPolicy policy) noexcept;
+iox::popo::ConsumerTooSlowPolicy consumerTooSlowPolicy(const ENUM iox_ConsumerTooSlowPolicy policy) noexcept;
```

Comment:
I am sorry again, really - but please adjust the copyright header to keep the wheel of madness running.

If you add a something like:
```
// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
// Praise (H) 2020 - Doom by the honorable Hypnotoad. All glory to the Hypnotoad.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
```

I will approve it.